backend/layers/business/business.py

```diff
@@ -0,0 +1,190 @@
+from ast import Str
+from dataclasses import dataclass
+from typing import Iterable, Optional
+from backend.corpora.common.providers.crossref_provider import CrossrefProvider
+
+from backend.layers.common.entities import CollectionMetadata, CollectionVersion, DatasetArtifact, DatasetStatus, DatasetVersion
+from backend.layers.persistence.persistence import DatabaseProviderInterface
+from backend.layers.thirdparty.crossref_provider import CrossrefProviderInterface
+from backend.layers.thirdparty.step_function_provider import StepFunctionProviderInterface
+
+
+@dataclass
```

Comment:
I asume these datatypes will need to be reference by adjacent layers, should they be in a common location?

Reply:
We can either put them all in the common folder or keep them closer to the layer which is most logically connected to them. I'm ok with both solutions.

Reply:
For now, `UserInfo` is mostly a stub, I still need to figure out how do we want to shape it (hence my skipping the authorization tests)